driver-core/src/main/java/com/datastax/driver/core/AbstractGettableData.java

```diff
@@ -21,8 +21,6 @@
 import java.nio.ByteBuffer;
 import java.util.*;
 
-import com.google.common.collect.Lists;
```

Comment:
Some changes are merely due to the merge from 2.0 that I did to update my branch. Sorry for that.